src/test/rgw/test_rgw_string.cc

```diff
@@ -12,6 +12,7 @@
  *
  */
 
+#include <errno.h>
```

Comment:
@cbodley 
I'm not sure what the project stanza is on this type of includes in include files.
Both types are present.